recipes/pycollo/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pycollo" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycollo-{{ version }}.tar.gz
+  sha256: 485d0d5d07b3ff3eba0d239c5ad6bdf96f65db8b2303d75438e1f49f9330a574
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script:
    - {{ PYTHON }} -m pip install . -vv
    - rm -rf $SP_DIR/tests
```

Reply:
@isuruf Thank you. Can you please advise why the `rm -rf $SP_DIR/tests` is required? Is this a side effect of my setting up my project incorrectly? E.g. including my `tests` directory in the project's MANIFEST.in?

Reply:
You need to exclude `tests` in https://github.com/brocksam/pycollo/blob/master/setup.py#L171. Otherwise they get installed into `lib/python3.9/site-packages/tests` which is not specifc to pycollo. This should be done fore pyproprop as well. The collision was because `pyproprop` and `pycollo` were both installing `tests/__init__.py` to the same location.

Reply:
Thank you, I'll make sure to update this in both packages for the next release.